docs/concepts/mixer.md

```diff
@@ -1,46 +1,200 @@
 
-**Quota Management**. Enables services to allocate and free quota on a number of dimensions, Quotas are used as a relatively simple resource management 
+- **Quota Management**. Enables services to allocate and free quota on a number of dimensions, Quotas are used as a relatively simple resource management 
 tool to provide some fairness between service consumers when contending for limited resources.
 
+These mechanisms are applied based on a set of attributes that are materialized for
+every request into the mixer. You can learn more about attributes [here]({{site.baseurl}}/docs/concepts/attributes.html).
+
+## Descriptors and policy objects
+
+Descriptors let you define the *shape* of the *policy objects* produced during the mixer's [attribute processing phase](#request-phases).
+In other words, descriptors let you control the set and type of values carried by individual policy objects. Some facts:
+
+- The mixer supports a variety of different descriptor kinds (`MetricDescriptor`, `LogEntryDescriptor`, `QuotaDescriptor`, etc)
+
+- For each descriptor kind, you can declare any number of descriptors within a given deployment.
+
+- While handling a request, the mixer's attribute processing phase can produce any number of policy objects for each of the
+configured descriptors.
+
+An example can help clarify the concepts. Let's consider the `MetricDescriptor` type which is defined as follows:
+
+    message MetricDescriptor {
+      string name = 1;
+      string display_name = 2;
+      string description = 3;
+    
+      enum MetricKind {
+        METRIC_KIND_UNSPECIFIED = 0;
+        GAUGE = 1;
+        COUNTER = 2;
+      }
+    
+      MetricKind kind = 4;
+      ValueType value = 5;
+      repeated LabelDescriptor labels = 6;
+    }
+
+Within a deployment configuration, you can declare a metric descriptor using a snippet of YAML:
+
+    metrics:
+      - name: request_count
+        kind: COUNTER
+        value: INT64
+        description: request count by source, target, service, and code
+        labels:
+        - name: source
+          value_type: STRING
+        - name: target
+          value_type: STRING
+        - name: service
+          value_type: STRING
+        - name: method
+          value_type: STRING
+        - name: response_code
+          value_type: INT64
+
+This is declaring a descriptor called request_count. Because of the kind and value fields, policy objects associated with this
+descriptor represent 64-bit integer counters. Additionally, each policy object is uniquely identified via the 5 listed labels.
+Producing a policy object for such a descriptor requires 6 pieces of information:
+
+- a 64-bit integer metric value 
+- a source string
+- a target string
+- a service string
+- a method string
+- a 64-bit integer response code
+
+Here is an example snippet of Istio configuratin which produces a policy object for the above descriptor:
+
+      metrics:
```

Comment:
Same, this is a single element list, not a single `MetricAspectConfigParams_Metric` (or whatever the type of that message inside of the aspect params is).